docs/sources/enterprise/license/activate-aws-marketplace-license/activate-license-on-eks.md

```diff
@@ -19,77 +19,89 @@ To activate your license, complete the following tasks:
+1. Deploy Grafana Enterprise on Amazon EKS.
 
-Update the deployment configuration to use Grafana Enterprise. Use `kubectl set image deployment/my-release grafana=grafana/grafana-enterprise:<version>` to update the container image to Grafana Enterprise version 8.3.0 or higher. For example, use `grafana/grafana-enterprise:8.3.3`.
+   For more information about deploying an application on Amazon EKS, refer to [Getting started with Amazon EKS – AWS Management Console and AWS CLI](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html).
 
-Versions of Grafana before 8.3.0 do not support licenses granted through AWS Marketplace.
+   For more information about installing Grafana on Kubernetes using the Helm Chart, refer to [Grafana Helm Chart](https://github.com/grafana/helm-charts/tree/main/charts/grafana#readme).
 
-## Task 2: Setup Grafana for high-availability
+1. Use `kubectl set image deployment/my-release grafana=grafana/grafana-enterprise:<version>` to update the container image to Grafana Enterprise version 8.3.0 or later.
 
-Update the [database]({{< relref "../../../../../administration/configuration.md#database" >}}) configuration section, so that Grafana use a shared database for storing dashboard, users, and other persistent data.
+   For example, enter `grafana/grafana-enterprise:8.3.3`.
 
-> If you do not have a MySQL database already, [create an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) and check the information needed to [connect to your RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_CommonTasks.Connect.html).
+> Only Grafana Enterprise versions 8.3.0 and later support licenses granted through AWS Marketplace.
 
-You can do this in one of two ways:
+## Task 2: Configure Grafana for high availability
 
-**Option 1:** Use `kubectl edit configmap grafana` to edit the Grafana configuration file. Under `grafana.ini` add the following section to your config:
+Grafana requires that you configure a database to hold dashboards, users, and other persistent data.
 
-```
-[database]
-type = mysql
-host = localhost:3306
-name = grafana
-user = [mysql database username]
-password = [mysql database password]
-```
+### Before you begin
 
-**Option 2:** Use `kubectl edit deployment my-release` to edit the pod environment variables. Under `env` add the following variables:
+- Ensure that you have a MySQL database available. For information about creating a MySQL database, refer to [Creating an Amazon RDS DB instance](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html).
```

Comment:
Same as for ECS, the database can by MySQL or Postgres.

Reply:
Done.